sqlmesh/core/analytics/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 import atexit
 import os
-import typing as t
+import typing_extensions as t
```

Comment:
Is this a correct import? Somehow I was sure that these extension were promoted to `typing` in later versions of Python

Reply:
we should revert this change

Reply:
reverted